fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -1470,6 +1468,67 @@ private void checkReplicaContinuousVersionSucc(long tabletId, Replica replica, b
         }
     }
 
+    private void checkReplicaContinuousVersionSucc(long tabletId, Replica replica, boolean alterReplicaLoadedTxn,
+            long minReplicaVersion, long maxReplicaVersion, List<PublishVersionTask> replicaPublishTasks,
+            Set<Long> errorReplicaIds, List<Replica> tabletSuccReplicas, List<Replica> tabletWriteFailedReplicas,
+            List<Replica> tabletVersionFailedReplicas) {
+        boolean failed = false;
+        for (PublishVersionTask task : replicaPublishTasks) {
+            if (task == null || !task.isFinished()) {
+                errorReplicaIds.add(replica.getId());
+                failed = true;
+            }
+        }
+        if (!failed) {
+            for (PublishVersionTask task : replicaPublishTasks) {
+                Map<Long, Long> backendSuccTablets = task.getSuccTablets();
+                // new doris BE will report succ tablets
+                if (backendSuccTablets != null) {
+                    if (!backendSuccTablets.containsKey(tabletId)) {
+                        failed = true;
+                        break;
+                    }
+                } else {
```

Comment:
delete 1491 ~ 1497,  because this code is just for compatibility and it's too old ( < 2.0.3),  the new BE should use new code. can delete this compatibility code then.